crates/usvg/src/parser/svgtree/parse.rs

```diff
@@ -284,40 +292,40 @@ pub(crate) fn parse_svg_element<'input>(
     let mut write_declaration = |declaration: &Declaration| {
         // TODO: perform XML attribute normalization
         if declaration.name == "marker" {
-            insert_attribute(AId::MarkerStart, declaration.value);
-            insert_attribute(AId::MarkerMid, declaration.value);
-            insert_attribute(AId::MarkerEnd, declaration.value);
+            insert_attribute(AId::MarkerStart, declaration.value, declaration.important);
```

Comment:
Too much copy-paste. Can we pass `declaration` directly? Or at least shorthand variables instead of typing `declaration.value, declaration.important` over and over again?

Reply:
Don't think it's possible to avoid since we are overriding the value in some cases, but I created a shorthand variable now.